apps/pollbook/backend/src/backup_worker_with_app.test.ts

```diff
@@ -0,0 +1,51 @@
+import { beforeEach, expect, test, vi } from 'vitest';
+import { err } from '@votingworks/basics';
+import { exportBackupVoterChecklist } from './backup_worker';
+import { withApp } from '../test/app';
+import { setupTestElectionAndVoters } from '../test/test_helpers';
+
+beforeEach(() => {
+  vi.clearAllMocks();
+});
+
+test.skip('exportBackupVoterChecklist returns an error when no USB drive is mounted', async () => {
+  await withApp(async ({ workspace, mockUsbDrive }) => {
+    mockUsbDrive.removeUsbDrive();
+    const result = await exportBackupVoterChecklist(
+      workspace,
+      mockUsbDrive.usbDrive
+    );
+    expect(result).toEqual(
+      err(new Error('No USB drive mounted, skipping backup'))
+    );
+  });
+});
+
+test.skip('exportBackupVoterChecklist returns an error when no election is configured', async () => {
+  await withApp(async ({ workspace, mockUsbDrive }) => {
+    mockUsbDrive.insertUsbDrive({});
+    const result = await exportBackupVoterChecklist(
+      workspace,
+      mockUsbDrive.usbDrive
+    );
+    expect(result).toEqual(
+      err(new Error('Machine not configured with election, skipping backup'))
+    );
+  });
+});
+
+test.skip('exportBackupVoterChecklist returns an error when no precinct is configured', async () => {
```

Comment:
Ended up skipping these because they flaked in CI, but holding out hope we'll have time to debug this soon

https://app.circleci.com/pipelines/github/votingworks/vxsuite/21431/workflows/82b68f39-1efe-48a5-94cb-148969e57954/jobs/935654